akka-projection-persistence-query-provider/src/main/scala/akka/projection/provider/TypedEventEnvelope.scala

```diff
@@ -0,0 +1,27 @@
+package akka.projection.provider
```

Comment:
Not so sure about this package name.
Maybe for the `TypedEventEnvelope`, we could keep it in `akka.persistence.query`